lib/contextual_logger.rb

```diff
@@ -133,25 +133,21 @@ def redactor
     end
 
     def format_message(severity, timestamp, progname, message, context: {})
-      message_hash = message_hash_with_context(severity, timestamp, progname, message, context: context)
-
       if @formatter
-        @formatter.call(severity, timestamp, progname, message_hash)
+        @formatter.call(severity, timestamp, progname, { message: ContextualLogger.normalize_message(message), **context })
       else
-        "#{message_hash.to_json}\n"
+        "#{basic_message_formatting(severity, timestamp, progname, message, context: context).to_json}\n"
```

Comment:
Do we still want `.to_json` here since that will escape the `<` `>` characters? (Vs. using `Oj`?)

Reply:
How about pushing the `to_json` part into that method too? That's possible now that that method isn't shared, and I think it will be a simpler contract. The method can be called `format_basic_json` or something like that.

Reply:
Moved the `.to_json` down into the method for a cleaner contract. I'd like to keep this gem as light as possible.  Best practice with contextual logging is to use your own formatter for structuring the log entry. This is just a very basic implementation.  The more dependencies that get added to the gem, the more conflicts and less plug and play it is.

Reply:
Make sense.